testnets/pockettestnet/chain.json

```diff
@@ -63,20 +61,21 @@
   },
   "images": [
     {
-      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg"
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.png"
```

Comment:
These image URLs are now not referencing a valid location. Just remove the /testnets/ part from them and we should be good.